graph_annotation.go

```diff
@@ -0,0 +1,18 @@
+package mackerel
+
+// GraphAnnotation parameters to post graph annotation
+type GraphAnnotation struct {
+	Service     string   `json:"service,omitempty"`
+	Roles       []string `json:"roles,omitempty"`
+	From        int64    `json:"from,omitempty"`
```

Comment:
hmm... it could be better to use `*time.Time` instead?

Users always have to call `*time.Time.Unix()` if we use int64 https://golang.org/pkg/time/#Time.Unix